components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml

```diff
@@ -109,6 +109,10 @@
   {{html clean="false"}}
     #set($valueListisEmpty = ($valueList.size() == 0))
     #set($xclass = $xwiki.getDocument($xclassName).getxWikiClass())
+    #set($ignored = {
+      'fields' : ['apgar5', 'negative_phenotype', 'negative_prenatal_phenotype'],
```

Comment:
Don't override the value for this parameter, since it's defined someplace else. Instead, modify the place where `ignored` is defined. It's in two places lower in this file.